js/ui/collection/ui.collection_widget.base.js

```diff
@@ -408,7 +416,10 @@ const CollectionWidget = Widget.inherit({
         this._updateFocusedItemState($target, true);
         this.onFocusedItemChanged(this.getFocusedItemId());
 
-        if(this.option('selectOnFocus')) {
+        const { selectOnFocus } = this.option();
+        const isDisabled = this._isDisabled($target);
```

Comment:
I see "isDisabled" -> I think about disabled state of the component -> I understand that it's unlikely that we check disabled state of the component -> I go to the variable initialization to check what is really disabled here.
Let's just make the name more complete?
```suggestion
        const isTargetDisabled = this._isDisabled($target);
```

Reply:
Yes, a good point! It was fixed here [25bf9f775185867becc632f8384161cc5d472868].